raisinbread/RB_files/RB_help.sql

```diff
@@ -1,5 +1,6 @@
 SET FOREIGN_KEY_CHECKS=0;
 TRUNCATE TABLE `help`;
 LOCK TABLES `help` WRITE;
+INSERT INTO `help` (`helpID`, `parentID`, `hash`, `topic`, `content`, `created`, `updated`) VALUES (123,12,'hash','BMI Instructions','The BMI calculator instrument can calculate the BMI and BMI Classification of a visitor. The two required fields for this instrument are the Date of Administration of the instrument and the Examiner name. The height and weight of the visitor must then be entered in either standard or metric units. The instrument will not accept an entry where the height and weight are given in both units. After this information is given, the instrument will calculate the BMI and BMI Classification of the visitor.',NULL,NULL);
```

Comment:
Alex, since this doesn't actually render for the BMI -- [ per your comment here ](https://github.com/aces/Loris/issues/6906#issuecomment-674946742)
I think it might not be because of the value in the `topic` column.  if you change it to `bmi` which i think would be the relevant test_name value for this instrument -- what happens? 

Could you also take another step here towards resolving it by checking the code and to see how the Help display/dropdown actually trying to locate relevant help text?  If it's not trying to match the `topic` column against a valid `test_name` value -- then what is it trying to look for?

Reply:
Yes I can definitely look into this :)

Reply:
@christinerogers @driusan  So after some digging:
The help file is handled by [helpHandler.js](https://github.com/aces/Loris/blob/main/htdocs/js/helpHandler.js). This then calls the [modules/help_editor/ajax/help.php](https://github.com/aces/Loris/blob/main/modules/help_editor/ajax/help.php) file with a GET request.
 
In THAT file, it first tries to render the help file by either using the module name (which is given by the request parameter `testName` or using the `subtest` parameter (in this case "bmi" or "aosi"). If it's not a module, it instead tries to load data from the `help` table using the request parameter `subtest` as the identifier. But when I went to an instrument data entry page and used the Developer Tool, I saw that when the GET request is sent to `help.php`, it only provides the 'testName` parameter in the request and doesn't provide a `subtest` value, so it never checks if there's a file in the help table and instead just looks for the markdown help file for the Instruments module. 

Let me know if this makes sense, I can explain it in more detail if necessary. I don't know how to fix this as of now. I can keep looking to see how to edit the GET request to provide the `subtest` parameter, but I haven't figured out where to do that yet.